lib/rspec/mocks/error_generator.rb

```diff
@@ -50,36 +50,35 @@ def raise_unexpected_message_error(message, *args)
       end
 
       # @private
-      def raise_unexpected_message_args_error(expectation, *args)
+      def message_error(expectation, *args)
```

Comment:
If I'm reading this right, `message_error` is a helper method, and not something meant to be called directly, right?  Can it be moved into the `private` section of the class to make that more explicit?

Also, `*args` is conventionally used for the list of arguments for one method call.  Here it looks like it's a list-of-argument-lists (unless I'm misreading the code), but the variable name doesn't reflect that.  If that's the case, I think `args_for_multiple_calls` is a better name.  This would apply to any other methods you're touching that similarly accept a list-of-argument-lists for multiple calls.

Thoughts?